pages/api/reset-password/index.ts

```diff
@@ -1,16 +1,11 @@
 import { ObjectId } from 'bson';
-import type { GetServerSidePropsContext, NextApiRequest, NextApiResponse } from 'next';
+import type { NextApiRequest, NextApiResponse } from 'next';
 import apiWrapper, { ValidObjectId, ValidType } from '../../../helpers/apiWrapper';
 import { logger } from '../../../helpers/logger';
-import redirectToHome from '../../../helpers/redirectToHome';
 import dbConnect from '../../../lib/dbConnect';
 import decodeResetPasswordToken from '../../../lib/decodeResetPasswordToken';
 import { UserModel } from '../../../models/mongoose';
 
-export async function getServerSideProps(context: GetServerSidePropsContext) {
```

Comment:
Should we keep this code? This is part of the password reset flow, so I think you should only need to access this page if you are not logged in.